pkg/kademlia/routing.go

```diff
@@ -321,3 +323,73 @@ func (rt *RoutingTable) ConnSuccess(ctx context.Context, node *pb.Node) {
 		zap.L().Debug("connection success error:", zap.Error(err))
 	}
 }
+
+func splitBucket(bIDs []bucketID, bitDepth int) ([]bucketID, []bucketID) {
+	var b0 []bucketID
+	var b1 []bucketID
+
+	for _, bID := range bIDs {
+		byteDepth := bitDepth / 8
+		bitOffset := bitDepth % 8
+		power := uint(7 - bitOffset)
+		bitMask := byte(1 << power)
+		b := bID[byteDepth]
+		if b&bitMask > 0 {
+			b1 = append(b1, bID)
+		} else {
+			b0 = append(b0, bID)
+		}
+	}
+	return b0, b1
+}
+
+// Prints the routing table graph as a dot graph in specified buffer
```

Comment:
The doc comment must start with the word `BufferedGraph`. Otherwise, the linter will keep complaining.